src/components/Settings.vue

```diff
@@ -17,6 +17,17 @@
             label="Networking"
           />
         </q-tabs>
+          <q-tabs
```

Comment:
Should there be another tabs group here?  Seems odd.

Reply:
fixed